examples/jsm/exporters/USDZExporter.js

```diff
@@ -461,7 +461,7 @@ function buildMaterial( material, textures ) {
 
 	}
 
-	if ( material.map !== null ) {
+	if ( material.map?.image ) {
```

Comment:
How about this: Instead of `material.map?.image`, let's keep `material.map !== null`. When the exporter later tries to access the image data, it reports a warning if no image data are defined and adds a default texture instead. This can be done by returning an empty 1x1 canvas in `imageToCanvas()`.

Try to serialize a texture with no image data is definitely wrong and something that should be reported as a warning.

Reply:
Agreed!

Reply:
The problem with creating a default texture is that then the intent of that texture would need to be passed in as well - is it a normal, should it be black or grey or white when the data doesn't exist?

Reply:
Unity has that "metadata" for each texture slot but I don't think that info currently lives anywhere in three, right? ("what's the invariant texture data for this slot")

Reply:
The idea of using default image data is to be able do export _something_. Alternatively, we have to throw an exception and abort the export entirely.

Please keep in mind that the 3D object is malformed to begin with when it has textures with empty image data. The app should not allow to export such assets in the first place.

Reply:
I'm fine with that, but then the FBXImporter and other cases where malformed textures can be produced by three.js should probably be fixed.

Reply:
Exporting something broken / wrong is **worse** than exporting nothing and throwing an exception; if there's no actual texture data I don't think a default texture should be constructed. Please disregard this PR entirely then and we just let it error as before.